### PR TITLE
feat : Static page for profile and post

### PR DESCRIPTION
**Pull request by anuj-kumary**
This PR includes:-

- Static Page For Profile and Post

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -4,4 +4,11 @@
     --main-bg-color: #1e2024;
     --btn-bg-color: #34343d;
     --btn-text-color: #fff;
+    --b3f: 250, 250, 250
+}
+
+body {
+    margin: 0;
+    /* background-color: #151e28; */
```
Comment (romabulani) [inline src/index.css]: Here you can remove the comment if not required

**Review by romabulani — APPROVED** [review]
- Liked that you have tried to make small components and reused them
- Modular structure of code
- Good to merge

File: src/index.css (hunk 1 above)
Comment (anuj-kumary) [reply src/index.css]: Done

**Review by Rohanmond — APPROVED** [review]
folder structure looks good , glad to see that you are using Material ui

**Review by PeeyushPrashant — APPROVED** [review]
clean & modular code
UI also looks good

**Pull request merged**